src/parse.c

```diff
@@ -29,9 +29,13 @@
 
 #include "parse.h"
 #include "util.h"
+#include "names.h"
 #include "error.h"
 #include "validation.h"
 
+#define NETPLAN_VERSION_MIN	2
+#define NETPLAN_VERSION_MAX	3
```

Comment:
We do we add these definitions here and keeping them in `parse.h` at the same time?

Reply:
These shouldn't be there, and are gone in V2 :)